go/install/install_osx.go

```diff
@@ -817,10 +849,10 @@ func installUpdater(keybaseBinPath string, force bool, keybaseVersion string, lo
 	return nil
 }
 
-func keybaseUpdaterPlist(label string, serviceBinPath string, keybaseBinPath string, keybaseVersion string) launchd.Plist {
+func keybaseUpdaterPlist(label string, serviceBinPath string, keybaseBinPath string) launchd.Plist {
 	plistArgs := []string{fmt.Sprintf("-path-to-keybase=%s", keybaseBinPath)}
 	envVars := DefaultLaunchdEnvVars(label)
-	comment := fmt.Sprintf("It's not advisable to edit this plist, it may be overwritten. This was installed by keybase version %s.", keybaseVersion)
```

Comment:
I thought it was a good idea but it causes the plist to change on every prerelease build (since the version changes)... and the install process sees this and does a re-install of the plist (since plist changed) which is unnecessary